### PR TITLE
Trainee main coursepage

### PR DESCRIPTION
**Pull request by codegracie**

### CODE REVIEW

**Comment by kian1213** [comment]
Ok

File: app/views/shared/_user_info.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <span class="label label-warning"><%= t ".text_skillshere" %></span>
+        <span class="label label-primary"><%= t ".text_skillshere" %></span>
+      </p>
+      <hr>
+      <strong><i class="fa fa-file-text-o margin-r-5"></i><%= t ".text_notes" %></strong>
+      <p><%= t ".text_noteshere" %></p>
+    </div>
+  </div>
+</div>
+
```
Comment (dieunb) [inline app/views/shared/_user_info.html.erb]: remove some white line

File: app/views/shared/_user_courselist.html.erb [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="box">
+              <div class="box-header with-border">
+                <h3 class="box-title">
+                  <i class="fa fa-folder-open-o"></i><%= t ".title_courses" %>
+                </h3>
+              </div>
+              <div class="box-body">
+                <table class="table table-bordered">
+                  <thead>
+                    <th style="width: 200px"><%= t 'name' %></th>
```
Comment (dieunb) [inline app/views/shared/_user_courselist.html.erb]: move style to css file

File: app/views/shared/_user_info.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+  <div class="box box-primary">
```
Comment (dieunb) [inline app/views/shared/_user_info.html.erb]: maybe it is missing open html tag `<div>`

Comment (codegracie) [inline app/views/shared/_user_info.html.erb]: already modified..

**Pull request merged**